frontend/src/app/main/ui/workspace/tokens/management/create/form.cljs

```diff
@@ -983,121 +1100,37 @@ custom-input-token-value-props: Custom props passed to the custom-input-token-va
-           (on-update-value e)))
-
-        input-props (mf/spread-props props {:default-value default-value
-                                            :on-update-value on-update-reference-value})]
-    [:div {:class (stl/css :typography-inputs-row)}
-     [:div {:class (stl/css :title-bar)}
-      [:div {:class (stl/css :title)}
-       (tr "labels.typography")]
-      [:& radio-buttons {:class (stl/css :listing-options)
-                         :selected (if reference-tab-active? "reference" "composite")
-                         :on-change on-toggle-tab
-                         :name "reference-composite-tab"}
-       [:& radio-button {:icon deprecated-icon/layers
-                         :value "composite"
-                         :title (tr "workspace.tokens.individual-tokens")
-                         :id "composite-opt"}]
-       [:& radio-button {:icon deprecated-icon/tokens
-                         :value "reference"
-                         :title (tr "workspace.tokens.use-reference")
-                         :id "reference-opt"}]]]
-     [:div {:class (stl/css :typography-inputs)}
-      (if reference-tab-active?
-        [:> typography-reference-input* input-props]
-        [:> typography-value-inputs* input-props])]]))
-
 (mf/defc typography-form*
   [{:keys [token] :rest props}]
-  (let [active-tab* (mf/use-state (if (cto/typography-composite-token-reference? (:value token)) :reference :composite))
-        active-tab (deref active-tab*)
-
-        custom-input-token-value-props
-        (mf/use-memo
-         (mf/deps active-tab)
-         (fn []
-           {:active-tab active-tab
-            :set-active-tab #(reset! active-tab* %)}))
-
-        ;; Remove the value from a stored token when it doesn't match the tab type
-        ;; We need this to keep the form disabled when there's an existing value that doesn't match the tab type
-        token
-        (mf/use-memo
-         (mf/deps token active-tab)
-         (fn []
-           (let [token-tab-type (if (cto/typography-composite-token-reference? (:value token)) :reference :composite)]
-             (cond-> token
-               (not= token-tab-type active-tab) (dissoc :value token)))))
-
-        on-get-token-value
+  (let [on-get-token-value
         (mf/use-callback
-         (fn [e prev-value]
+         (fn [e prev-composite-value]
            (let [token-type (obj/get e "tokenType")
                  input-value (dom/get-target-val e)
                  reference-value-input? (not token-type)]
              (cond
                reference-value-input? input-value
 
-               (empty? input-value) (dissoc prev-value token-type)
-               :else (assoc prev-value token-type input-value)))))]
-    [:> form*
+               (empty? input-value) (dissoc prev-composite-value token-type)
+               :else (assoc prev-composite-value token-type input-value)))))
+
+        update-composite-backup-value
+        (mf/use-callback
```

Comment:
We prefer to use always `use-fn` (this is a synonym). Please replace it in other places too.